projects/entity-framework.md

```diff
@@ -1,27 +1,20 @@
 # Entity Framework
 
-Entity Framework is the recommend data access technology for new applications in .NET. It is an object-relational mapper that enables .NET developers to work with relational data using domain-specific objects. It eliminates the need for most of the data-access code that developers usually need to write.
-
-### Entity Framework has two versions: 
-* Entity Framework 6, for .NET Full Framework (Windows)
-* Entity Framework Core, for .NET Core
-
-[Entity Framework 6](https://github.com/aspnet/EntityFramework6) is the legacy Entity Framework for .NET Full Framework, that runs only under Windows Systems.
-
-[Entity Framework Core](https://github.com/aspnet/EntityFrameworkCore) is the new version for .NET Core applications running Windows, Linux and MAC OS System. This version is being recreated, for better performance and portability. 
+[Entity Framework (EF) Core](https://github.com/aspnet/EntityFrameworkCore) is the recommend data access technology for new applications in .NET. Entity Framework (EF) Core is a lightweight and extensible version of the popular Entity Framework data access technology. EF Core is an object-relational mapper (O/RM) that enables .NET developers to work with a database using .NET objects. It eliminates the need for most of the data-access code that developers usually need to write.
 
 ## Project Details
-* [Project Info Site](https://github.com/aspnet/EntityFramework/wiki)
-* [Project Code Site](https://github.com/aspnet/EntityFramework)
-* Project License Type: [Apache 2.0](https://github.com/aspnet/EntityFrameworkCore/blob/master/LICENSE.txt)
+* [Project Info Site](https://github.com/aspnet/EntityFrameworkCore/wiki)
```

Comment:
Not sure our wiki is ideal for "project info site". It really has very few things in it. My first intuition is to just remove it, but I noticed all projects have "project info site". So, maybe point to the same thing as the "project code site" or point to the documentation?